charts/vault-secrets-operator/templates/cluster-role-binding.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if and .Values.rbac.create (eq .Values.vault.authMethod "kubernetes") }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: {{ if not .Values.rbac.namespaced -}} Cluster {{- end -}} RoleBinding
```

Comment:
Hmm, is this what we want? This will still create a `RoleBinding` instead of a `ClusterRoleBinding` if `rbac.namspaced` is true.

How will a `ClusterRoleBinding` be created? Maybe I'm missing something.

Reply:
Ah thanks, this should always be a `ClusterRoleBinding`.